setup.cfg

```diff
@@ -13,3 +13,6 @@ ignore =
 
 [upload_sphinx]
 upload-dir = doc/_build/html
+
+[pytest]
+norecursedirs = .git build dist docs
```

Comment:
It would be nice to *only* look in `nengo` for tests, since right now it is the only directory with tests, but I can't find a way to do this.